File: python/day_one/level_two/task_one.py


def check_most_appeared_value(numbers):

    new_list = []
    
    for number_one in numbers:

        count = 0
        
        for number_two in numbers:
                 
            if number_one == number_two:
                count+=1
                
                
        if count > 1 and number_one not in new_list:
            new_list.append(number_one)
    return new_list

File: java/dayOne/levelTwo/TaskOne.java
import java.util.Arrays;

public class TaskOne {

    public static int [] checkMostAppearedValue(int [] numbers){
        
            
           int [] mostAppearedNumber = new int [numbers.length];
   
                 int index = 0;
            for(int count = 0; count < numbers.length; count++){
                             
                for(int countTwo = count+1; countTwo< numbers.length; countTwo++){
            
                    if(numbers[count] == numbers[countTwo]){
                       mostAppearedNumber[index] = numbers[count]; 
                        index ++;
                        
                        }
                        
                    }
                     
                 }
                 int [] result = Arrays.copyOf(mostAppearedNumber, index); 
                 
                 return result;
                                 
    }




}

